master/usage/migrating-pools.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Changing IP pools
+---
+
+## About changing IP pools
+
+When using {{ site.prodname }} IPAM, each workload is assigned an address from the selection of configured IP pools.
+There are a few reasons you may want to change the IP pool being used on a running cluster, including moving to a
+larger CIDR which can accomodate more workloads, or moving off of a CIDR that was used accidentally.
```

Comment:
"accomodate" --> "accommodate"

I think the last sentence could be tightened up a little bit. For example, it might benefit from a more active construction, less words, and perhaps a bulleted list format. Use your best judgement, but I might suggest something more like:

```
You may want to modify the IP pool of a running cluster for one of the following reasons.
- To move to a larger CIDR that can accommodate more workloads.
- To move off of a CIDR that was used accidentally.
```